draco/analysis/sidereal.py

```diff
@@ -249,26 +249,33 @@ def process(self, data):
 class SiderealStacker(task.SingleTask):
     """Take in a set of sidereal days, and stack them up.
 
-    This will apply relative calibration.
+    Also computes the variance over sideral days using an
+    algorithm that updates the sum of square differences from
```

Comment:
It came up recently that is would be nice to start documenting the algorithms that are used in the code. In this case, just mentioning a name or a link would be helpful.

Reply:
Sounds good, I've added a reference to the relevant paper.